universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java

```diff
@@ -172,14 +187,18 @@ protected ContainerTag renderProgramApplicationTitleAndProgressIndicator(
   }
 
   /** Returns whole number out of 100 representing the completion percent of this program. */
-  private int getPercentComplete(int blockIndex, int totalBlockCount) {
+  private int getPercentComplete(int blockIndex, int totalBlockCount, boolean forSummary) {
     if (totalBlockCount == 0) return 100;
     if (blockIndex == -1) return 0;
 
     // Add one to blockIndex for 1-based indexing, so that when applicant is on first block, we show
     // some amount of progress.
     // Add one to totalBlockCount so that when applicant is on the last block, we show that they're
     // still in progress. Save showing "100% completion" for the application review page.
-    return (int) (((blockIndex + 1.0) / (totalBlockCount + 1.0)) * 100.0);
+
+    double numerator = forSummary ? blockIndex : blockIndex + 1;
```

Comment:
(blocking) Can you add a comment for why these two numbers are different in the `forSummary` case? Unclear to me at the moment.

PS this computation may change in the near future pending guidance from UX, e.g., we might not add 1 to the total, since it makes the progress bar kind of weird / inaccurate with respect to the number counts in the top right corner.

Reply:
Added to the comment about why the 1s are added.

Reply:
Thanks!

One more question I have is the vocab ("summary view", `forSummary`, etc). The review page with the submit button currently has the title "Application summary". I wonder if we should use different verbiage to differentiate.

I have no idea what though...thinking

Reply:
Maybe like `isCompleted` to represent whether the form is completed (in which case we show the submit view).

Reply:
Nvm - we discussed offline and I was just confused.